babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -992,7 +992,7 @@ private void checkParseInfixCast(String sqlType) {
 
   @Test public void testCurrentTimestampFunctionBigQuery() {
     final String sql = "select current_timestamp";
```

Comment:
Can we delete all of the BigQuery-specific DATE/TIME tests since they are unparsing without parens now?